docs/_includes/generated/netty_shuffle_environment_configuration.html

```diff
@@ -30,7 +30,7 @@
             <td><h5>taskmanager.network.pipelined-shuffle.compression.enabled</h5></td>
```

Comment:
please also copy the PR description to the commit message - in the PR description you explained the `isCopied` problem in much more detail :)

Reply:
Thanks for the review and comments, I agree that we can improve or replace the current pipeline compression implement in the future if we have a better solution and the change is simple enough (only need to remove several lines of code). Besides, the added config option is simple enough and we can keep the config option unchanged or map it to new value even when we replace the current implementation in the future.